### ISSUE TITLE
Add search by Genre/popular. I remember old app had this feature

### ISSUE THREAD

**Issue by ghost** [deceptions/no#118]
![IMG_20220130_211925](https://user-images.githubusercontent.com/88455631/151701553-071a4a23-7762-4b2c-a229-78767473b805.jpg)
Can you make this thing disappear it's unecessary. Or replace it with episode title. But fix the word written it's not placed properly.

**Comment by Adi-MayNotSpeak** [comment]
> ![IMG_20220130_211925](https://user-images.githubusercontent.com/88455631/151701553-071a4a23-7762-4b2c-a229-78767473b805.jpg) Can you make this thing disappear it's unecessary. Or replace it with episode title. But fix the word written it's not placed properly.

Disagree , it's helpful for me

**Comment by ghost** [comment]
> 

Why disagree is it you don't know what source are you on? If so just click that source button below and you'll know. Man that thing is covering the screen.

**Comment by Adi-MayNotSpeak** [comment]
> > 
> 
> Why disagree is it you don't know what source are you on? If so just click that source button below and you'll know. Man that thing is covering the screen.

Yeah it could be shorten to just the quality in the corner but not remove it
